### DISCUSSION TITLE
Problemi di connessione con sky Wifi

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#1934] [HUB: Home Assistant]
Ciao a tutti, 

ho installato HA core (su rasp pi3b, al momento su scheda SD) a casa dei miei genitori, dove hanno sky Wifi. 
Non ho terminato la configurazione dell'accesso da remoto in quanto non ci sono riuscito e francamente non serve, era uno sfizio.
Home assistant continua ad avere perdite di connessioni, rallentamenti, a volte non si riesce a raggiungere la pagina internet di HA.

Se provo a pingare il rasp, mi esce errore"Host di destinazione non raggiungibile".
Credo siano problematiche legate a SkyWifi, avete dei suggerimenti da darmi?

Grazie

**Comment by KC-inDomus** [comment]
> Ciao a tutti,
> 
> ho installato HA core (su rasp pi3b, al momento su scheda SD) a casa dei miei genitori, dove hanno sky Wifi. Non ho terminato la configurazione dell'accesso da remoto in quanto non ci sono riuscito e francamente non serve, era uno sfizio. Home assistant continua ad avere perdite di connessioni, rallentamenti, a volte non si riesce a raggiungere la pagina internet di HA.
> 
> Se provo a pingare il rasp, mi esce errore"Host di destinazione non raggiungibile". Credo siano problematiche legate a SkyWifi, avete dei suggerimenti da darmi?
> 
> Grazie

in LAN non dovresti avere alcuno dei problemi che citi. è connesso via ethernet o wifi?

**Reply by calcioscacchi** [reply]
Wifi. Provo in lan
